src/app/utils/accountPageUtils.js

```diff
@@ -4,6 +4,43 @@ import axios from 'axios';
 import appConfig from '../data/appConfig';
 import { CLOSED_LOCATION_REGEX } from '../data/constants';
 
+/**
+ * Swap actual status labels for something more patron-friendly
+ */
+export const swapStatusLabels = (html) => {
+  html = html.replace(/<td class="patFuncStatus"> AVAILABLE <\/td>/g, '<td class="patFuncStatus"> REQUEST PLACED </td>')
+  html = html.replace(/<td class="patFuncStatus"> READY SOON <\/td>/g, '<td class="patFuncStatus"> READY FOR PICKUP </td>')
+  return html
+}
+
+function returnOnlyTable (html) {
+  // for some reason the patFuncTitle div comes back with variable amounts of spaces
+  // apparently dependent on the text inside of it. 
+  // const hasTitle = html.match(/<div([\s].*)class="patFuncTitle">([\s\S]*?)<table border="0" class="patFunc">([\s\S]*?)<\/table>/gm)
+  // const table = html.match(/<table border="0" class="patFunc">([\s\S]*?)<\/table>/)
+  // if (hasTitle) {
+  //   return hasTitle[0]
+  // } else if (table) {
+  //   return table[0]
+  // } else throw new Error('Webpac html is not formatted as expected')
+  throw new Error('Webpac html is not formatted as expected')
```

Comment:
Is this commented out for testing purposes?

This function should also have a unit test.